sql/usuario.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE usuario (
+    id int primary key autoincrement,
+    user varchar(20)
+    pass varchar(40)
```

Comment:
pass debe ser not null, @CaroSosa para que te fijes como es la tabla usuario! para poderlo insertar